File: FASE 1/PHP/Secuanciales/PlayXbox.php
<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Document</title>
</head>
<body>
    <form action ="PlayXbox.php" method="post">
        <label>Ingrese la cantidad de PlayStation 5 vendidos: </label>
        <input type="number" id="play" name="play" required><br>

        <label>Ingrese la cantidad de Xbox vendidos: </label>
        <input type="number" id="xbox" name="xbox" required><br>
        <button type="submit">Enviar</button>
    </form>
</body>
</html>

<?php
if($_POST){
    $pesoplay=(float)2.3; $pesoxbox=(float)3.2; $costoenvio=5;
    $play=$_POST['play']; $xbox=$_POST['xbox'];
    $pesopaquete=(($play*$pesoplay)+($pesoxbox*$xbox));
    $costototal=($pesopaquete*$costoenvio);
    echo 'El peso total de el paquete son: ' . $pesopaquete . ' libras.<br>';
    echo 'El costo total de el envio son : $' . $costototal;
 }
?>

File: FASE 1/Python/Secuanciales/PlayXbox.py
pesoplay=2.3
pesox_box=3.2
costo_envio=5
playvendidos=int(input("Ingrese los PlayStation 5 vendidos: "))
xboxvendidos=int(input("Ingrese los Xbox Serie S vendidos: "))
pesopaquete=((playvendidos*2.3)+(xboxvendidos*3.2))
costototal=pesopaquete*costo_envio
print(f"El peso total de el paquete son: {pesopaquete} Libras")
print(f"El total del envio son: ${costototal}")